### ISSUE TITLE
Search sources are shown twice in the component at the preview and runtime

### ISSUE THREAD

**Issue by aparveen** [Esri/solutions-components#182]
Test App: https://sg-em-qaext.mapsqa.arcgis.com/apps/instant/notification/index.html?appid=94a0f5cd859d496585b55449a407d78e

![image](https://user-images.githubusercontent.com/10148401/219207040-c678ead6-319f-473f-b80c-72a5d4e78abe.png)

**Comment by aparveen** [comment]
Can't repro this anymore, Yay

**Comment by aparveen** [comment]
Seeing this again in multiple apps in devext and QA.
it looks like it shows the default search source list twice, the one I configured shows as expected (only once.

**Comment by aparveen** [comment]
I am not seeing this in QA this morning but another issue, after I add a search source the app preview got blank and showing a console error
Test app: https://sg-em-qaext.mapsqa.arcgis.com/apps/configure-template/index.html?appid=c68e7014f43a4e4b97caadb8ec251759
![image](https://user-images.githubusercontent.com/10148401/219970805-e918f994-3411-4c7b-9482-96f2e1ad0178.png)

**Comment by jmhauck** [comment]
@aparveen I believe they have a patch for this coming in from the IA side. We should test again tomorrow.

**Comment by aparveen** [comment]
This is fixed
